apollo-router/tests/integration/telemetry/mod.rs

```diff
@@ -1,5 +1,9 @@
+#[cfg(any(not(feature = "ci"), all(target_arch = "x86_64", target_os = "linux")))]
```

Comment:
Is it related to the collector jaeger thing ? If yes, why not using collector in every cases ?

Reply:
It's just we weren't running these tests on OSX before.